### PR TITLE
19285 - Update allowable actions for Continuation In

### PR DESCRIPTION
**Pull request by ketaki-deodhar**
*Issue #:* /bcgov/entity#19285

*Description of changes:*

- Add continuation in allowable action
- Update unit tests

Updated unit tests are successfully running:

image

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

**Review by jamespaologarcia — APPROVED** [review]
LGTM

File: legal-api/src/legal_api/core/meta/filing.py [hunk 1]
```diff
@@ -254,6 +254,22 @@ class FilingTitles(str, Enum):
...
+            'BC': 'BC Limited Company Continuation In Application',
+            'BEN': 'BC Benefit Company Continuation In Application',
+            'CC': 'BC Community Contribution Company Continuation In Application',
+            'ULC': 'BC Unlimited Liability Company Continuation In Application',
+        },
+        'codes': {
+            'BC': 'CONTI',
+            'BEN': 'CONTI',
+            'ULC': 'CONTI',
+            'CC': 'CONTI'
```
Comment (severinbeauvais) [inline legal-api/src/legal_api/core/meta/filing.py:270]: Are these the fee codes? Does this cover off bcgov/entity#19284?

Comment (severinbeauvais) [inline legal-api/src/legal_api/core/meta/filing.py:270]: At what point do we start using the new legal types (C, CBEN, CUL, CCC)?

Comment (ketaki-deodhar) [reply legal-api/src/legal_api/core/meta/filing.py:270]: I referenced to ticket https://app.zenhub.com/workspaces/entities---olga-65af15f59e89f5043c2911f7/issues/gh/bcgov/entity/19283

Comment (vysakh-menon-aot) [reply legal-api/src/legal_api/core/meta/filing.py:270]: I will update it in my ticket

**Review by vysakh-menon-aot — APPROVED** [review]

File: legal-api/src/legal_api/core/meta/filing.py (hunk 1 above)
Comment (severinbeauvais) [reply legal-api/src/legal_api/core/meta/filing.py:270]: Can 19284 be closed then?

Comment (ketaki-deodhar) [reply legal-api/src/legal_api/core/meta/filing.py:270]: Yes. I think so. I can add reference to this PR

**Review by severinbeauvais — APPROVED** [review]

**Pull request merged**